Vagrantfile

```diff
@@ -129,10 +129,10 @@ Vagrant.configure("2") do |config|
     end
   end
 
-    config.vm.define "dev" do |nodeconfig|
-      nodeconfig.vm.box = "ubuntu/impish64"
+    config.vm.define "elastic-agent" do |nodeconfig|
+      nodeconfig.vm.box = "ubuntu/jammy64"
 
-      nodeconfig.vm.hostname = "elastic-agent-dev"
+      nodeconfig.vm.hostname = "elastic-agent-dev.elastic.dev.internal"
```

Comment:
This change is useful for testing the FQDN feature using the `elastic-agent` Vagrant VM.